HPXMLtoOpenStudio/measure.rb

```diff
@@ -1447,6 +1450,9 @@ def self.add_foundation_walls_slabs(runner, model, building, spaces)
                                 drywall_thick_in, film_r, mat_ext_finish)
       end
     end
+    if fndwall_slab_size < fndwall_size
+      fail "There existing foundation wall(s) not attached to any slab. Please double check."
```

Comment:
Can we give a more specific error message? Something like: "Foundation wall 'foo' is adjacent to 'basement - unconditioned' but no corresponding slab was found adjacent to 'basement - unconditioned'".